code/game/objects/items/robot/items/tools.dm

```diff
@@ -1,4 +1,5 @@
 #define PKBORG_DAMPEN_CYCLE_DELAY (2 SECONDS)
+#define POWER_RECHARGE_CYBORG_DRAIN_COEFFICIENT (0.4 KILO WATTS)
```

Comment:
It's kinda strange that a coefficient uses units, but also the recharge variable is used in multiple places. If both instances of recharging scales by time, then I guess we could just put units on the energy and power variables.

If it's not, then I'm not sure what to do to keep it clean, other than removing the coefficient and just using separate variables instead I guess.

Reply:
I'll rename it from coefficient to multiplier so we don't have to debate about terminology